gradle.properties

```diff
@@ -2,7 +2,7 @@ android.useAndroidX=true
 android.enableJetifier=true
 kotlin.code.style=official
 GROUP=dev.mobile
-VERSION_NAME=1.19.6-SNAPSHOT
+VERSION_NAME=1.20.0-SNAPSHOT
```

Comment:
according to documentation - should it be changed to non-snapshot version instead?

Reply:
Yep. Fixed it.